client/test/TestFluid.res

```diff
@@ -2802,6 +2787,226 @@ let run = () => {
       "\"fia~x\"",
     )
   })
+  describe("And/Or", () => {
```

Comment:
In a blank handler, enter `|` three times. You end up in a surprising place. This isn't new (just checked prod) but might be worth adding a test case around this, or an issue to fix this. Or maybe this is somehow expected?
![Peek 2022-12-13 16-52](https://user-images.githubusercontent.com/906686/207451918-cdaa4e38-5bd7-434f-a27e-2d1ca2b7eb87.gif)

Reply:
I suppose it's expected, and I agree it's a bit weird. What would a better behaviour be?

Reply:
If my code is `&&|` (where `|` is my cursor),
then I'd expect `&` input to update the code to `___ && ___ &|& ___`